### ISSUE TITLE
Change bst code to use header stack

### ISSUE THREAD

**Issue by hesingh** [P4ELTE/use_cases#2]
I would abandon t4p4s and just use the new p4lang/p4c/backends/dpdpk which supports header stack.

Also, my company's (closed, not open-sourced) P4toVPP p4c backend also supports header stack.

**Comment by gpz76** [comment]
Hi,

Isn’t this backend the same that was presented by Intel at the P4 WS? If yes, we believe it has a different design: it is based on the packet framework while t4p4s uses native DPDK calls. But you have a point anyway, so we’ll initiate a discussion with the owners.
Thanks!



From: Hemant Singh ***@***.***>
Sent: Sunday, June 6, 2021 6:42 PM
To: P4ELTE/use_cases ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [P4ELTE/use_cases] Change bst code to use header stack (#2)


I would abandon t4p4s and just use the new p4lang/p4c/backends/dpdpk which supports header stack.

**Comment by hesingh** [comment]
Yes,  the p4c dpdk backend was presented at the 2021 P4 WS.  This backend supports header stack and thus you could use this new backend - thanks.
